### ISSUE TITLE
Wrong handle

### ISSUE THREAD

**Issue by zeke** [bevacqua/twitter-for-github#8]
https://github.com/wilsonpage links to https://twitter.com/LJHarb

![screen shot 2016-07-14 at 5 21 01 pm](https://cloud.githubusercontent.com/assets/2289/16859883/89ed56d2-49e7-11e6-9703-ac983dc05fc1.png)

cc @ljharb @wilsonpage

**Comment by ljharb** [comment]
Did you click to his from mine?

**Comment by zeke** [comment]
Nope. Clicked from https://github.com/electron/electron/pull/6490

**Comment by zeke** [comment]
Another one:

![screen shot 2016-10-23 at 11 29 52 pm](https://cloud.githubusercontent.com/assets/2289/19635644/b15be842-9978-11e6-832c-b1c4b09c2570.png)

cc @jlfwong @hughsk

**Comment by zeke** [comment]
🤔

**Comment by jlfwong** [comment]
I... don't really need to be notified about a Chrome extension I don't use incorrectly reporting information about me.

Though I do appreciate that I was someone's page you tested?

**Comment by bevacqua** [comment]
@zeke That's basically just because the underlying detection lib is kind of flaky. Honestly don't have time to work on this though.

**Comment by zeke** [comment]
@jlfwong sorry to disturb. Just pinging the related parties in case it leads to clues about possible causes of the failure.

**Comment by bevacqua** [comment]
@zeke I think root cause is that there's a package somewhere that infers email addresses from commits, and sometimes that can lead to related contributors' emails being used instead of the actual person's email.

**Comment by ljharb** [comment]
If it uses "committer" instead of "author", that could easily happen.

**Comment by benoror** [comment]
Same here!

<img width="242" alt="screen shot 2017-02-28 at 23 38 40" src="https://cloud.githubusercontent.com/assets/119117/23447225/105591cc-fe0f-11e6-8fdf-28f44224fe44.png">

**Comment by bnb** [comment]
Found another incorrect handle: 
![image](https://user-images.githubusercontent.com/502396/29036171-521d1064-7b6c-11e7-9e60-5b73dd4047db.png)
